gr-blocks/python/blocks/qa_file_metadata.py

```diff
@@ -57,26 +57,17 @@ def test_001(self):
 
         handle = open(outfile, "rb")
         header_str = handle.read(blocks.parse_file_metadata.HEADER_LENGTH)
-        if(len(header_str) == 0):
-            self.assertFalse()
-
-        try:
-            header = pmt.deserialize_str(header_str)
-        except RuntimeError:
-            self.assertFalse()
+        self.assertGreater(len(header_str), 0)
 
+        header = pmt.deserialize_str(header_str)
```

Comment:
No `try` / `except` is required here. If an exception is raised, the test will exit immediately and report that.

Reply:
Same goes for three more instances below.